doc/source/user_guide/ug_case_settings.rst

```diff
@@ -8,7 +8,52 @@ Case Settings
 There are two important files that define the case, **cice.settings** and 
 **ice_in**.  **cice.settings** is a list of env variables that define many
 values used to setup, build and run the case.  **ice_in** is the input namelist file
-for CICE.  Variables in both files are described below.
+for CICE.  Variables in both files are described below.  In addition, the first
+table lists available preprocessor directives to activate or deactivate various
```

Comment:
```suggestion
table lists available preprocessor macros to activate or deactivate various
```

I think the right terminology is "preprocessor macro" (https://gcc.gnu.org/onlinedocs/cpp/Object-like-Macros.html#Object-like-Macros). "`#ifdef`", "`#ifndef`" and "`#define`" are preprocessor directives.

(there are other instances, I'm just pointing it out here.)